boring/src/ssl/mod.rs

```diff
@@ -110,71 +110,71 @@ bitflags! {
     /// Options controlling the behavior of an `SslContext`.
     pub struct SslOptions: c_uint {
         /// Disables a countermeasure against an SSLv3/TLSv1.0 vulnerability affecting CBC ciphers.
-        const DONT_INSERT_EMPTY_FRAGMENTS = ffi::SSL_OP_DONT_INSERT_EMPTY_FRAGMENTS;
+        const DONT_INSERT_EMPTY_FRAGMENTS = ffi::SSL_OP_DONT_INSERT_EMPTY_FRAGMENTS as _;
 
         /// A "reasonable default" set of options which enables compatibility flags.
-        const ALL = ffi::SSL_OP_ALL;
+        const ALL = ffi::SSL_OP_ALL as _;
```

Comment:
What type it currently generates for `#define`'s?

Reply:
The default type is `i32` (unless a constant requires `i64`).

That can be changed to `u32` or even to the smallest type that can represent a given value, but `i32` is what requires the least amount of conversions.